app/controllers/admin/order_cycles_controller.rb

```diff
@@ -235,5 +235,14 @@ def require_order_cycle_set_params
     def ams_prefix_whitelist
       [:basic, :index]
     end
+
+    def order_cycle_params
+      return params[:order_cycle] if params[:order_cycle].empty?
```

Comment:
You've added this in a couple of places. Why it is it needed?

Reply:
some of the tests send empty params[:order_cycle] and the following require/permit statement fails.